src/mmw/js/src/modeling/templates/scenarioToolbarTabContent.html

```diff
@@ -2,8 +2,12 @@
 
 <div id="modification-btn-wrapper"> <!-- Modifications Dropdown -->
     <div class="dropdown"> <!-- Dropdown Button -->
+        {% if shapes|length == 0 %}
```

Comment:
Well, try it, and if it doesn't work this is fine. Perhaps we can do `===` to be even more explicit.